opencti-platform/opencti-front/src/private/components/riskAssessment/risks/RiskTrackingPopover.js

```diff
@@ -487,12 +529,13 @@ class RiskTrackingPopover extends Component {
                           name="related_responses"
                           fullWidth={true}
                           size="small"
+                          multiple={true}
                           variant='outlined'
                           style={{ height: '38.09px' }}
                           containerstyle={{ width: '100%' }}
                         >
                           {riskStatusResponse.map((value, i) => (
```

Comment:
@sachin755 same here, is it necessary to check if `riskStatusResponse` is null or will it always at the least be an empty array?

Reply:
@DallasP9124  it'll have an array of objects or an empty array.